task-2/src/components/ErrorDisplay404/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+// import { Link } from 'react-router-dom';
+
+export const ErrorDisplay404 = () => {
+	return (
+		<div>
+			<p>Page Not Found</p>
+			<p>404</p>
+			{/* <Link to='/'>
```

Comment:
It's a good functionality, let's uncomment it.
A side note - avoid leaving commented code without a GOOD reason in production.

Reply:
Resolved.